editor/data-binding/lists.mdx

```diff
@@ -0,0 +1,97 @@
+
+![Image](/images/editor/add-artboard-list.png)
+
+Once the Artboard List is added to your hierarchy, you can select it and see its inspector.
+
+![Image](/images/editor/artboard-list-property.png)
+
+The next step is to bind data to it using DataBinding. This will determine the content and number of items in your List. There are 2 ways to generate List content:
+
+- Using the View Model List property
+- Using a View Model Number property together with a Number to List converter
+
+## View Model List Property
+
+<Note>Before continuing, it's important to understand the fundamentals of DataBinding, in particular, what a View Model is, how to create one and how to bind it to an object's properties. Learn more in the [DataBinding Overview](/editor/data-binding/overview).</Note>
+
+A View Model List is a property that can contain N items which each represent a View Model instance. In order to be used in a List, the View Model must be bound to an Artboard.
+
+To Create a View Model List and bind it to an Artboard List:
+
+1. Navigate to the Data tab in the Editor
+2. Click the + button beside View Models to create a View Model (this will represent your List item)
+3. Bind that View Model to an Artboard that you want to use to render your List item. This is where you may also want to create additional View Model properties and bind those to object properties on that Artboard.
+4. Click the + button beside View Models to create another View Model (this will be the View Model that contains your List and should be bound to the Artboard where you want your List to render)
+5. Select the newly created View Model and click the + button next to it
+6. From the popout select **List**. This adds a List property to the View Model
+![Image](/images/editor/list-property.png)
+7. Select the List property and in the inspector on the right, you can add items by clicking Add List item
+![Image](/images/editor/list-items.png)
+8. Once a List item is added, you can click the settings button to the left of its name in order to set the View Model and View Model instance for that item
+9. After adding your List items, go back to the Hierarchy tab, select the Artboard List, and from the Artboard List property dropdown, select the ViewModel List property you created above. 
+10. Run the state machine and you should see your list items. Remember that the layout will be determined by the Artboard List's parent, so modify the parent's settings in order to tweak your List's layout
+
+<Note>Rive's runtimes will provide APIs to modify the List and List items at runtime (for example, adding or removing items), however this is not yet available in Early Access.</Note>
+
+## View Model Number with Converter
+
+The second way to populate your list is by specifying the number of items you want in your list along with the ViewModel (Artboard) you want to instance. This can be done using a View Model Number property in combination with the new Number to List converter.
```

Comment:
```suggestion
The second way to populate your list is by specifying the number of items you want in your List along with the ViewModel (Artboard) you want to instance. This can be done using a View Model Number property in combination with the new Number to List converter.
```